src/utils/ColorPicker.js

```diff
@@ -1154,7 +1154,7 @@
     var duringDragEvents = {};
     duringDragEvents['selectstart'] = prevent;
     duringDragEvents['dragstart'] = prevent;
-    duringDragEvents['touchmove mousemove'] = move;
+    duringDragEvents['mousemove'] = move;
```

Comment:
Which is the reason of this?

Reply:
The Event object provided by Touchmove don't meet our needs ( or require new implementations ), you can try your demo on emulator or touch device to look the result .

The Mousemove event is fired after Touchmove and the Event object provided is properly implemented .

So removing Touchmove the ColorPicker is working as well .